app/backend/pyproject.toml

```diff
@@ -1,12 +1,29 @@
-[tool.black]
+[tool.ruff]
 line-length = 120
 
-[tool.isort]
-skip_gitignore = true
-include_trailing_comma = true
-multi_line_output = 3
-line_length = 120
-known_first_party = ["couchers", "proto"]
+[tool.ruff.lint]
+select = [
+    "E",   # pycodestyle
+    "F",   # pyflakes
+    "UP",  # pyupgrade
+    "B",   # flake8-bugbear
+    "C4",  # flake8-comprehensions
+    "A",   # flake8-builtins
+    "I",   # isort
+]
+ignore = [
+    "B007",   # Loop control variable not used within loop body
+    "E501",   # Line too long
+    "E711",   # Comparison to `None` should be `cond is None`
+    "E712",   # Avoid equality comparisons to `True`; use `if approved:` for truth checks
+    "F811",   # Redefinition of unused variable
+    "F841",   # Local variable is assigned to but never used
+    "UP015",  # Unnecessary open mode parameters
+]
```

Comment:
Is this still an accurate list? These linters rules seem useful, are we failing a lot of them?

Reply:
To `B007`: You can put an underscore in front of the variable that is not required, but named. E.g: `for topic, _name, items in group:`.

Reply:
@HansBambel Just saw that you commented, great to see you here 😄